js/components/tab.js

```diff
@@ -18,6 +18,8 @@ const dnd = require('../dnd')
 const windowStore = require('../stores/windowStore')
```

Comment:
If it's all good with you, I might try moving tab.js later. I tried moving it now but it has a lot of deps to move around.